src/components/Panel/Panel.stories.tsx

```diff
@@ -1,8 +1,16 @@
 import { ComponentStory } from "@storybook/react";
 import styled from "styled-components";
 
+import Ticket from "../../interfaces/Ticket";
 import Panel from "./Panel";
 
+const items: Ticket[] = [
```

Comment:
[nitpick] - this variable's name could be something like `mockItems`. Super trivial so don't worry though😄